### ISSUE TITLE
Error: Attempted a typeid of NULL pointer!

### ISSUE THREAD

**Issue by ChrissiQ** [sass/libsass#825]
I'm using libsass with grunt-sass 0.17.0 and node-sass 1.2.3.  I'm trying to port some code from ruby sass and compass, to libsass and bourbon, on Windows.

I've come to this one which is a stumper.

Error: Attempted a typeid of NULL pointer!

There is no other information in this error.  Is there a way to show a stack trace or anything... even just a line number.  If this is all there is, it's going to be hard to debug this.

**Comment by xzyfer** [comment]
You're best bet would be compile your sass files with Libsass directly to
find out which file it crashes on. Libsass should give some kind of error
message. You can use gdb to get more information.

Alternatively you could hold out for grunt-sass to update to Libsass 3.1.0.
They're waiting on node-sass which is currently in 2.0.0-beta so it
shouldn't be long now.

**Comment by xzyfer** [comment]
I'm closing this issue since there has been no reply. Feel free to re-open if the above doesn't help.

**Comment by ChrissiQ** [comment]
I was not able to run with libsass directly.  But I found out through other means what the problem was.

For anyone coming here for future reference:

`Error: Attempted a typeid of NULL pointer!` seemed to occur when a variable has not yet been defined and tries to reference itself.  i.e.

`$font-family-sans-serif: "Open Sans", #{$font-family-sans-serif};`

This code compiled in ruby sass because $font-family-sans-serif was previously defined, but did not compile in libsass because of modifications I had to make to get libsass to work, ended up removing the code that had previously defined $font-family-sans-serif for me.  So when I tried to reference it, it was null.

The error message was decidedly unhelpful without a line number, but with a line number, it would have made some sense.  :)
